### PR TITLE
submitting Homework 1

### PR DESCRIPTION
**Pull request by belasobral93**

### CODE REVIEW

File: bela-belasobral93/Homework-1/99-bottles-of-beer/app.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+  
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else). 
+*/
+
+for(var i=99; i > 0; i--){
```
Comment (AurielleP) [inline bela-belasobral93/Homework-1/99-bottles-of-beer/app.js:13]: The new version of JavaScript, ES6, introduces let in favor of var; this new type of variable keeps scope, which works great in loops and conditionals.

File: bela-belasobral93/Homework-1/temp-converter/js/main.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+*/
+
+
+ //return first element in document with id "temperature", in this case, the user input
+
+let celcius;
+let temp = document.querySelector("#temperature").value;
+
+   function fToC(){
+      return celcius = (temp - 32) / 1.8;
```
Comment (AurielleP) [inline bela-belasobral93/Homework-1/temp-converter/js/main.js]: You want to try to make sure that your return statement is the last thing that runs in your function. If you return before you console log it wont ever get to that step

File: bela-belasobral93/Homework-2/object-homework/app.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  // - make noise (each pet makes its own unique noise, as specified by `noise`.
+
+
+// Answer Starts Here
+let some_pet = {
+  Pet: {
+    petName: ["leah", "xuxu", "gucci"],
+    petSpecies: ["cat", "dog", "hamster"] 
+    petBreed: ["maltese", "siamese", "teddy"]
+    petNoise:
```
Comment (AurielleP) [inline bela-belasobral93/Homework-2/object-homework/app.js]: Where is the petNoise function?

File: bela-belasobral93/Homework-2/object-homework/app.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  // - a noise that it can make (e.g. 'bark', 'meow', etc.)
+  // Every pet can:
+  // - make noise (each pet makes its own unique noise, as specified by `noise`.
+
+
+// Answer Starts Here
+let some_pet = {
+  Pet: {
+    petName: ["leah", "xuxu", "gucci"],
+    petSpecies: ["cat", "dog", "hamster"]
```
Comment (AurielleP) [inline bela-belasobral93/Homework-2/object-homework/app.js]: Each property should have a comma at the end before the next one

**Comment by belasobral93** [comment]
@AurielleP i noticed my subsequent homework submissions (2 and 3) are automatically going into my original "submitting Homework 1"  pull request....

**Pull request merged**